_positions/epa-fellow-otaq-technical-architect.md

```diff
@@ -0,0 +1,53 @@
+
+We are looking for an IT Solutions Architect to help us modernize the Engines and Vehicles Compliance Information System (EV-CIS) which has over 19,000 registered users from 13 industries and 38 countries and many stakeholders and partners including engine and vehicle manufacturers, external federal/state agencies such as DOE, DOT (co-regulators), Treasury, DHS and the California Air Resource Board (co-regulator).
+
+You will provide solutions architecture and technical leadership using your familiarity with HTML, Java (or other object-oriented coding) and Oracle database experience to assist in moving the EPA to a more user-centric, modular approach to systems development.
+
+Your challenge is to help the EV-CIS program to transition from legacy technology stacks to more modern technologies. You will work to understand the needs and the desires of the agency, design systems to meet those needs and work cross functionally to implement those solutions. By doing so, you’ll greatly increase EPA’s overall capacity to spend scarce IT budget resources on innovative projects, improving the digital experience for the public.
+
+##Key Objectives:
+**You will serve as the innovative technical leader in the modernization efforts for the the highly complex and significant Engines and Vehicles Compliance Information System (EV-CIS).**
+- Establish and lead an EV-CIS technical integrated project team consisting of all EV-CIS developers to share information and to define the current software architecture
+- Architect system to be as simple as possible, elegantly delivering on client need and embracing opportunities for improvement through re-engineering or refactoring
+- Analyze current system architecture and existing products (developed by numerous external vendors and internal teams since 2006) to identify the functionality and capabilities that can be leveraged and areas requiring refactoring to develop an integration plan to create a seamless technology ecosystem with a consistent user experience
+- Manage the development of  EV-CIS with an API first mindset using reusable components and operational strategies leveraging cloud services to support scalability, availability, performance, monitoring, backup, etc, to improve the security, resiliency, and agility of infrastructure and applications
+- Ensure strategic alignment of technical design/architecture to meet business growth and direction
+- Drive planning and delivery by utilizing agile tools such as product roadmaps, backlogs, measurable success criteria, and write user stories
+
+**You will promote the adoption of modern technology practices through teaching, coaching, and knowledge sharing.**
+- Deliver educational workshops to accelerate learning and adoption of agile development and DevOps principles and practices
+- Inspire honesty and integrity through open communication and liberal knowledge sharing across a multi-disciplinary teams
```

Comment:
@rooneywp Which should this be, `a multi-disciplinary team` or `multi-disciplinary teams`?